<launch>
	<param name="move_rate" value="0.5" />
  	<param name ="maxForce" value="100" />

	<node name="linear_acc" pkg="sub8_diagnostics" type="linear_acc" />
</launch>
